### DISCUSSION TITLE
Managing size of modal editor view

### DISCUSSION THREAD

**Discussion by dmoskalenko-datagrok** [epam/ketcher#1284] [Q&A]
Is there in Ketcher Reaction option for managing the size of Editor form, 
currently, the form is too big?
Example
<img src="https://user-images.githubusercontent.com/98885254/154522615-c99196c4-2ac3-416a-9e84-56a629876372.png" width="400" />

**Comment by AndreiMazol** [comment]
Dear @dmoskalenko-datagrok ,

The size of Ketcher Editor form is managed by parent DOM element. for details please look  at https://github.com/epam/ketcher/discussions/583#discussioncomment-841139

Regards,
Andrei
